vxfreeswitch/voice.py

```diff
@@ -162,12 +171,15 @@ def onBackgroundJob(self, ev):
             response, content = ev.rawresponse.split()
             if response == "+OK":
                 d.callback(content)
+            else:
+                d.errback(ev)
```

Comment:
``.errback`` should probably be passed an exception or a Failure instance instead of ``ev``. Perhaps the exception can have ``.ev`` as an attribute if we need access to it for some reason?